db/crud.go

```diff
@@ -1617,6 +1618,10 @@ func (db *DatabaseContext) assignSequence(ctx context.Context, docSequence uint6
 			if err = db.sequences.releaseSequence(ctx, docSequence); err != nil {
 				base.WarnfCtx(ctx, "Error returned when releasing sequence %d. Falling back to skipped sequence handling.  Error:%v", docSequence, err)
 			}
+			numSkippedSequences := doc.Sequence - docSequence
+			if numSkippedSequences > oversizeSkippedSequenceWarning {
+				base.WarnfCtx(ctx, "doc %s / previous rev: %s new rev: %s has sequence %d which is significantly ahead of the known highest sequence %d", base.UD(doc.ID), doc.CurrentRev, newRevID, doc.Sequence, docSequence)
```

Comment:
It's not immediately obvious to me what 'known highest sequence' means here.  Can we change this to 'next allocated sequence'?